	With this patch you can rename user-commands/xindy.v2.in
	=> user-commands/xindy.v2 and simplify user-commands/Makefile.am
	accordingly.

diff -ur -x Makefile.in -x aclocal.m4 -x configure -x autom4te.cache xindy.orig/user-commands/texindy xindy/user-commands/texindy
--- xindy.orig/user-commands/texindy	2009-03-22 12:09:29.000000000 +0100
+++ xindy/user-commands/texindy	2009-11-24 22:03:22.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#! /usr/bin/env perl
 # $Id: texindy,v 1.8 2009/03/22 11:08:18 jschrod Exp $
 #------------------------------------------------------------
 # (history at end)
diff -ur -x Makefile.in -x aclocal.m4 -x configure -x autom4te.cache xindy.orig/user-commands/xindy.in xindy/user-commands/xindy.in
--- xindy.orig/user-commands/xindy.in	2009-03-29 13:14:55.000000000 +0200
+++ xindy/user-commands/xindy.in	2009-11-24 22:03:21.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#! /usr/bin/env perl
 # $Id: xindy.pl,v 1.13 2009/03/29 11:14:04 jschrod Exp $
 #------------------------------------------------------------
 # (history at end)
diff -ur -x Makefile.in -x aclocal.m4 -x configure -x autom4te.cache xindy.orig/user-commands/xindy.v2.in xindy/user-commands/xindy.v2.in
--- xindy.orig/user-commands/xindy.v2.in	2005-05-30 23:10:02.000000000 +0200
+++ xindy/user-commands/xindy.v2.in	2009-11-24 22:03:35.000000000 +0100
@@ -1,4 +1,4 @@
-#!@XINDY_PERL@
+#! /usr/bin/env perl
 # $Id: xindy.in,v 1.14 2005/05/02 19:16:27 jschrod Exp $
 #------------------------------------------------------------
 #
